pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -317,7 +317,7 @@ func (jc *MPIJobReconciler) onOwnerCreateFunc() func(event.CreateEvent) bool {
 		}
 
 		jc.Scheme.Default(mpiJob)
-		msg := fmt.Sprintf("MPIJob %s/%s is created.", mpiJob.Namespace, e.Object.GetName())
+		msg := fmt.Sprintf("MPIJob %s is created.", e.Object.GetName())
```

Comment:
I modified this form to make uniform other frameworks since only here shows the namespace when created.